src/Api.js

```diff
@@ -33,6 +33,31 @@ class Api {
 		] );
 	}
 
+	/**
+	 * Get parsed edit summary for the given revision.
+	 * @param {number} revId
+	 * @return {Promise} Resolving Object with keys 'comment' and 'size'.
+ 	 */
+	fetchEditSummary( revId ) {
+		return this.mwApi.ajax( {
+			action: 'compare',
+			fromrev: revId,
+			torelative: 'prev',
+			prop: 'parsedcomment|size',
+			formatversion: 2
+		} ).then( data => {
```

Comment:
Formattingwise, I think it'll be easier to read with a newline:
```
} ).then(
    data => { 
       ...
    },
    failData => {
      ...
    }
);
```

But, granted, it might just be a personal preference :)

Reply:
That does look prettier! I wonder if there's an eslint option for this